$NetBSD: patch-Makefile,v 1.4 2025/08/16 01:20:27 ryoon Exp $

* Workaround for rebuild in install stage.
* Do not install share/ in no-check-symbols case.
* Disable symbol comparison, it's fragile

--- Makefile.orig	2025-08-08 15:50:22.000000000 +0000
+++ Makefile
@@ -885,9 +885,9 @@ finish: check-symbols
 endif
 endif
 
-install: finish
+install:
 	mkdir -p "$(INSTALL_DIR)"
-	cp -p -r "$(SYSROOT)/lib" "$(SYSROOT)/share" "$(SYSROOT)/include" "$(INSTALL_DIR)"
+	cp -p -r "$(SYSROOT)/lib" "$(SYSROOT)/include" "$(INSTALL_DIR)"
 
 DEFINED_SYMBOLS = $(SYSROOT_SHARE)/defined-symbols.txt
 UNDEFINED_SYMBOLS = $(SYSROOT_SHARE)/undefined-symbols.txt
@@ -1018,7 +1018,7 @@ check-symbols: $(STARTUP_FILES) libc
 
 	# Check that the computed metadata matches the expected metadata.
 	# This ignores whitespace because on Windows the output has CRLF line endings.
-	diff -wur "$(EXPECTED_TARGET_DIR)" "$(SYSROOT_SHARE)"
+	#diff -wur "$(EXPECTED_TARGET_DIR)" "$(SYSROOT_SHARE)"
 
 
 ##### BINDINGS #################################################################
